guides/generate_punks_with_gan.md

```diff
@@ -0,0 +1,164 @@
+
+*Note!* Our model needs an input tensor of dimensions 100x1x1 for to do a single inference, or (BatchSize)x100x1x1 for generating a batch of images. In this demo we'll start by generating 4 punks at a time.
+
+## Step 3 — Creating a Gradio interface
+
+At this point you can even run the code you have with `predict(<SOME_NUMBER>)`, and you'll find your freshly generated punks in your file system at `./punks.png`. To make a truly interactive demo, though, we'll build out a simple interface with Gradio. Our goals here are to:
+
+* Set a slider input so users can choose the "seed" value
+* Use an image component for our output to showcase the generated punks
+* Use our `predict()` to take the seed and generate the images
+
+With `gr.Interface()`, we can define all of that with a single function call:
+
+```python
+import gradio as gr
+
+gr.Interface(
+    predict,
+    inputs=[
+        gr.inputs.Slider(label='Seed', minimum=0, maximum=1000, default=42),
+    ],
+    outputs="image",
+).launch()
+```
+
+## Step 4 — Even more punks!
+
+Generating 4 punks at a time is a good start, but maybe we'd like to control how many we want to make each time. Adding more inputs to our Gradio interface is as simple as adding another item to the `inputs` list that we pass to `gr.Interface`:
+
+```python
+gr.Interface(
+    predict,
+    inputs=[
+        gr.inputs.Slider(label='Seed', minimum=0, maximum=1000, default=42),
+        gr.inputs.Slider(label='Number of Punks', minimum=4, maximum=64, step=1, default=10), # Adding another slider!
+    ],
+    # ...etc.
+```
+
+The new input will be passed to our `predict()` function, so we have to make some changes to that function to accept a new parameter:
+
+```python
+def predict(seed, num_punks):
+    torch.manual_seed(seed)
+    z = torch.randn(num_punks, 100, 1, 1)
+    punks = model(z)
+    save_image(punks, "punks.png", normalize=True)
+    return 'punks.png'
+```
+
+When you start up your Gradio app again, you should see a second slider that'll let you control the number of punks!
+
+## Step 5 - Polishing it up
+
+Your Gradio app is pretty much good to go, but you can add a few extra things to really make it ready for the spotlight ✨
+
+We can add some examples that users can easily try out by adding this to the `gr.Interface`:
+
+```python
+gr.Interface(
+    # ...
+    # keep everything as it is, and then add
+    examples=[[123, 15], [42, 29], [456, 8], [1337, 35]],
+).launch(cache_examples=True) # cache_examples is optional
+```
+
+The `examples` parameter takes a list of lists, where each item in the sublists is ordered in the same order that we've listed the `inputs`. So in our case, `[seed, num_punks]`. Give it a try!
+
+You can also try adding a `title`, `description`, and `article` to the `gr.Interface`. Each of those parameters accepts a string, so try it out and see what happens 👀 `article` will also accept HTML, which we'll explore in a future guide!
+
+----------
```

Comment:
FYI this gets rendered as slightly more space. not sure if that's what's intended.
<img width="931" alt="Screen Shot 2022-04-19 at 12 10 55 AM" src="https://user-images.githubusercontent.com/9021060/163885619-3fa654d7-8619-474a-a880-bca5a6a2852a.png">

Reply:
Yup! I just copied the format used in some of the other guides (e.g. https://github.com/gradio-app/gradio/blob/main/guides/image_classification_in_pytorch.md?plain=1#L87)